docs/padgett/visualization_JN_usecase/usecase.md

```diff
@@ -0,0 +1,114 @@
+# Visualization of spatially distributed data 
+## <center> **Jupyter notebook for visualization of spatially distributed data in risk and resilience analysis** </center>
+**Raul Rincon - Dept. of Civil and Environmental Engineering, Rice University** <br>
+**Jamie E. Padgett - Dept. of Civil and Environmental Engineering, Rice University** <br>
+<br>
+Key Words: visualization; risk and resilience; infrastructure systems; static, interactive, and animated maps and figures, effective communication
+
+## Resources 
+The examples presented make use of the following DesignSafe resources: <br>
+[Jupyter notebooks on DS Juypterhub](https://www.designsafe-ci.org/rw/workspace/#!/Jupyter::Analysis)<br>
```

Comment:
Please point to specific use cases in Jupyter  and could use links like these: https://deploy-preview-45--ds-use-case-preview.netlify.app/brandenberg/apiusecases/usecase/

Reply:
Thanks! We have modified the usecase.md to solve the two comments